### DISCUSSION TITLE
Allow change of the recpie folder path/name for all user and new users to come

### DISCUSSION THREAD

**Discussion by jeremyy44** [nextcloud/cookbook#1088] [Ideas and discussions]
Basicly I would want the option to change the default path of my recipe folder from /Recipies to /.Recipies for all my users at the same time and for it to also apply to newly created users which would allow for a invisible folder which is in my opinion a cleaner experience for a user

**Comment by christianlupus** [comment]
This is a duplicate of #666 mostly. At least related.

**Reply by jeremyy44** [reply]
> This is a duplicate of #666 mostly. At least related.

Yes it is a duplicate my mistake.
